test/test_model.py

```diff
@@ -54,7 +54,19 @@ def test_relations_keys(self):
         fake_script(self, 'relation-ids',
                     """[ "$1" = db2 ] && echo '["db2:5", "db2:6"]' || echo '[]'""")
         fake_script(self, 'relation-list',
-                    """([ "$2" = 5 ] && echo '["remoteapp1/0", "remoteapp1/1"]') || ([ "$2" = 6 ] && echo '["remoteapp2/0"]') || exit 2""")
+                    """
+case "$2" in
+    5)
+        echo '["remoteapp1/0", "remoteapp1/1"]'
+        ;;
+    6)
+        echo '["remoteapp2/0"]'
+        ;;
+    *)
+        exit 2
+    ;;
+esac
+""")
```

Comment:
definitely an improvement, though hopefully testing_harness can clean this up even more.